EDA.qmd

```diff
@@ -278,7 +254,7 @@ If you've encountered unusual values in your dataset, and simply want to move on
       filter(between(y, 3, 20))
     ```
 
-    We don't recommend this option because just because one measurement is invalid, doesn't mean all the measurements are.
+    We don't recommend this option because, for a given observation, just because the value for one variable is invalid doesn't mean the values for all variables are.
```

Comment:
```suggestion
    We don't recommend this option because one invalid value doesn't imply that all the other values for that observation are also invalid.
```